SPECS/malaga/malaga.spec

```diff
@@ -17,15 +16,13 @@ Patch0:         malaga-rename-map_file.diff
 # linking anymore
 Patch1:         malaga-malshow-lm.patch
 Patch2:         malaga-aarch64.patch
-
```

Comment:
Couldn't add a comment in the right spot, so adding it here:

Could I ask you to change the extension of `malage-rename-map_file.diff` to `.patch`? It is not one of the `SourceX` files, it's definitely used as a patch, so we should do that to be consistent with our other specs. We should then also remove it from the signatures JSON btw.